erddap/templates/private/dataset.deployment.xml

```diff
@@ -301,184 +301,35 @@
         </addAttributes>
     </dataVariable>
 
+    {% for req_var in reqd_qc_vars %}
     <dataVariable>
-        <sourceName>time_qc</sourceName>
-        <destinationName>precise_time_qc</destinationName>
+        <sourceName>{{req_var}}</sourceName>
+        <destinationName>{{dest_var_remaps.get(req_var, req_var)}}</destinationName>
```

Comment:
Perhaps I should have added a comment here.  This tries to get any name remappings and defaults to the original variable name if not found.  defaultdict didn't seem to cover this use case elegantly, but maybe I was missing something.